source/manual/development-disk-space.html.md

```diff
@@ -30,24 +30,24 @@ On the host machine, you can safely delete:
 
 On the VM, you can safely delete old Elasticsearch indexes:
 
-  ```
-  cd /var/govuk/govuk-puppet/development-vm/replication
-  bundle exec ruby close_and_delete_old_indices.rb
-  ```
+```shell
+$ cd /var/govuk/govuk-puppet/development-vm/replication
+$ bundle exec ruby close_and_delete_old_indices.rb
+```
 
 If you plan on replicating all your data again, you can also delete big mongodb
 databases to clear space.
 
-```
-mongo
+```shell
+$ mongo
```

Comment:
I think the platform health team were talking about replication for dev machines, as some people's machines are filling up. Maybe ask them to take a look?